roles/openshift_control_plane/tasks/main.yml

```diff
@@ -129,6 +129,12 @@
   # copy to this location to bypass initial bootstrap request
   - /etc/origin/node/node.kubeconfig
 
+- name: Pre-pull Origin image
+  command: "atomic pull {{ osm_image }}"
```

Comment:
Maybe use a ternary for a variable here for docker/atomic.

Reply:
I'm guessing it should be ternary for `--storage`, as atomic would use docker (or cri-o if available) to pull

Reply:
Potentially, but CI complaining 'atomic' no such file or directory.